src/cata_tiles.h

```diff
@@ -84,6 +85,7 @@ const std::unordered_map<std::string, TILE_CATEGORY> to_TILE_CATEGORY = {
     {"hit_entity", TILE_CATEGORY::HIT_ENTITY},
     {"weather", TILE_CATEGORY::WEATHER},
     {"overmap_terrain", TILE_CATEGORY::OVERMAP_TERRAIN},
+    {"overmap_weather", TILE_CATEGORY::OVERMAP_TERRAIN},
```

Comment:
Looks like both terrain and weather strings are using the same category here. Is that intentional?